core/src/test/java/org/apache/accumulo/core/file/blockfile/cache/TestCachedBlockQueue.java

```diff
@@ -58,34 +106,38 @@ public void testQueue() {
     long expectedSize = cb1.heapSize() + cb2.heapSize() + cb3.heapSize() + cb4.heapSize()
         + cb5.heapSize() + cb6.heapSize() + cb7.heapSize() + cb8.heapSize();
 
-    assertEquals(queue.heapSize(), expectedSize);
+    assertEquals(expectedSize, queue.heapSize());
 
-    LinkedList<org.apache.accumulo.core.file.blockfile.cache.lru.CachedBlock> blocks =
-        queue.getList();
-    assertEquals(blocks.poll().getName(), "cb1");
-    assertEquals(blocks.poll().getName(), "cb2");
-    assertEquals(blocks.poll().getName(), "cb3");
-    assertEquals(blocks.poll().getName(), "cb4");
-    assertEquals(blocks.poll().getName(), "cb5");
-    assertEquals(blocks.poll().getName(), "cb6");
-    assertEquals(blocks.poll().getName(), "cb7");
-    assertEquals(blocks.poll().getName(), "cb8");
+    List<org.apache.accumulo.core.file.blockfile.cache.lru.CachedBlock> blocks = getList(queue);
 
+    assertEquals(Arrays.asList("cb1", "cb2", "cb3", "cb4", "cb5", "cb6", "cb7", "cb8"),
```

Comment:
Alternatively, could use `List.of()` instead of `Arrays.asList()` throughout the test.

Reply:
Addressed in d0b5f0d3cb